ryanoasis/nerd-fonts
# Added chocolatey install scripts

#### Description

Chocolatey package generation scripts. Added a few fonts that I use. Rest can be added as described in the README.

#### Requirements / Checklist

- [] Scripts execute without error (if necessary):
  - If any of the scripts were modified they have been tested and execute without error, e.g.:
    - `./font-patcher Inconsolata.otf --fontawesome --octicons --pomicons`
    - `./gotta-patch-em-all-font-patcher\!.sh Hermit`

#### What does this Pull Request (PR) do?

Adds chocolatey packages and respective generating scripts.

#### How should this be manually tested?

- Run the `generate_packages.py` to generate chocolatey packages for the fonts.
- Go inside any generated folder and run `choco install <font-id> -s .`
    - e.g: `choco install hasklig-nerd-font -s .`
- Fonts should get installed successfully

#### Any background context you can provide?

Chocolatey is a package manager for windows. Easy installation of fonts using the commands like `choco install hasklig-nerd-font`.

#### What are the relevant tickets (if any)?

#223 

#### Screenshots (if appropriate or helpful)
